docs/guides/admin/docs/configuration/https/self-signed-certificates.md

```diff
@@ -0,0 +1,112 @@
+Self-signed certificates
+========================
+
+Since commit [4225bf](https://github.com/opencast/opencast/commit/4225bf90af74557deaf8fb6b80b0705c9621acfc)
+([security advisory GHSA-44cw-p2hm-gpf6](https://github.com/opencast/opencast/security/advisories/GHSA-44cw-p2hm-gpf6))
+Opencast services check the validity of the certificates presented by
+third parties and when connecting to each other remotely.
+
+The validity check is basically performed the same way as any other HTTPS client
+would do. Some of the steps involve:
+* Validate the host name
+* Look up if the certificate is signed by a trusted Certificate Authority (CA)
+
+In case of self-signed certificates, a check whether the certificate is signed
+by a trusted CA, would fail.
+
+You are advised to obtain a valid certificate, issued by a trusted CA like
+[Let's Encrypt](https://letsencrypt.org/).
+
+However, valid certificates are not always an option for testing or developer
+instances of Opencast, especially with Enterprise Firewalls or CAA records in
+place.
+
+Generating self-signed certificates
+-----------------------------------
+
+A self-signed certificate for multiple host names can be created with openSSL
+as follows:
+
+`testing-cert.req.conf`:
+
+```
+[req]
+distinguished_name = req_distinguished_name
+x509_extensions = v3_req
+prompt = no
+[req_distinguished_name]
+C = CH
+ST = Wallis
+L = Matterhorn
+O = Apereo
+OU = Opencast
+CN = opencast
+[v3_req]
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+extendedKeyUsage = serverAuth
+subjectAltName = @alt_names
+[alt_names]
+DNS.1 = opencast
+DNS.2 = presentation.local
+DNS.3 = admin.local
+DNS.4 = admin
+DNS.5 = presentation
+DNS.6 = worker
+DNS.7 = worker.local
+```
+
+```
+openssl req \
+  -x509 \
+  -nodes \
+  -days 365 \
+  -newkey rsa:2048 \
+  -keyout key.pem \
+  -out testing.pem \
+  -config testing-cert.req.conf \
+  -extensions 'v3_req'
+```
+
+In order to view the just generated cert:
+
+```
+openssl x509 -in testing.pem -text -noout
+```
+
+Trusting self-signed certificates
+---------------------------------
+
+In order to use self-signed certificates for testing or developer instances of
+Opencast, import your self-signed certificate(s) into the Java Trust Store
+(bundle of trusted CA certs) and restart Opencast.
+
+1. Store your certificate in a format compatible with `keytool` somewhere:
+
+```
+cat >/tmp/testing.crt <<EOL
+-----BEGIN CERTIFICATE-----
+MIIGJzCCBA+gAw...
+...O6g==
+-----END CERTIFICATE-----
+EOL
+```
```

Comment:
This is purely Markdown formatting. You are basically adding a list with a single item then end the list, start a code block and start all over again. Some Markdown renderers will even render it with the list item 1 every time. Instead, make the code block part of the list item:

```suggestion
       cat >/tmp/testing.crt <<EOL
       -----BEGIN CERTIFICATE-----
       MIIGJzCCBA+gAw...
       ...O6g==
       -----END CERTIFICATE-----
       EOL

```

The same goes for the other items below.

Reply:
Good catch. Used Atom's built-in preview.

Reply:
Unfortunately, you need empty lines around the code. Otherwise, it won't render correctly.

```
1. something

       some code

2. some other thing
```

---

1. something

       some code

2. some other thing

Reply:
Done